CONTRIBUTING.md

```diff
@@ -95,7 +95,7 @@ OneUI's [release procedure](RELEASING.md) and [changelog](CHANGELOG.md) automati
 [optional footer]
 ```
 
-Commit types include:
+Commit types (required) include:
```

Comment:
I find it a bit confusing... we have the PR title set like this. Do we need to repeat the PR title in the squash commit message? I guess only one of these is indeed necessary. Would be good to clarify which.

Reply:
When squashing and merging a PR in Github the title is transferred to the squash and merge dialog. So if your PR title already contains what you want your (squashed) commit title to be you should be good (apart from the body / footer). Still, you need to be careful and double check that yourself, as with everything. We can document every single thing but in general you should always be careful about what you do.

Reply:
I don't think that's correct. See my PR yesterday. It transferred the text of the commit to the squash, which wasn't named according to convention.

Reply:
I'm pretty sure that's correct because it has worked for me that way each time I squashed and merged PRs.

![squash-merge](https://user-images.githubusercontent.com/798175/60251174-55179200-98c8-11e9-8fae-381b3e659424.gif)

So something else must have gone wrong in your particular case. Either way we should always be careful with what we do no matter to which extent we document things.

Reply:
please note that you have the same message for your commit as for the PR title.

Reply:
Well, that's exactly what I expect